automatic/pdfcreator/pdfcreator.nuspec

```diff
@@ -23,10 +23,27 @@ PDFCreator easily creates PDFs from any Windows program. Use it like a printer i
 
 ## Features
 
-- Convert your Documents to PDF, JPG, PNG, TIF and more
-- Merge multiple documents to one file
+- Convert your documents to PDF, JPG, PNG, TIF and PDF/A
+- Merge and arrange multiple documents to one file
 - Profiles make frequently used settings available with one click
 - Use automatic saving to have a fully automated PDF printer
+- Secure against opening, printing or modifying your PDFs with a password
+- Support for digital signatures
+- COM interface for application or scripted actions
+- PDFCreator is Open Source software and licensed under the terms of the Affero General Public License (AGPL)
+- You are free to use PDFCreator at home or at work
+- You may access the source code and compile it on your own, as long as it stays under the AGPL
+
+### Install Options
```

Comment:
Please change this to `## Package Parameters`

Reply:
See [1.2.3](https://github.com/chocolatey/chocolatey-coreteampackages/blob/master/CONTRIBUTING.md#123-informative-description) please for details.

Reply:
OK, so not '## Package Parameters`, but `## Notes`.